LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Grant Goodyear
```

Comment:
Add other contributors here, maybe? I license my contributions to this project under CC0, so I don't actually need acknowledgement.